src/cdk/a11y/aria-describer/aria-describer.ts

```diff
@@ -218,10 +218,21 @@ export class AriaDescriber implements OnDestroy {
 
   /** Determines whether a message can be described on a particular element. */
   private _canBeDescribed(element: Element, message: string): boolean {
-    return element.nodeType === this._document.ELEMENT_NODE && message != null &&
-        !!`${message}`.trim();
+    if (!this._isElementNode(element)) {
+      return false;
+    }
+
+    const trimmedMessage = message == null ? '' : `${message}`.trim();
```

Comment:
Unnecessary template string literal? Also, omit the `== null`? 
```ts
message ? '' : message.trim();
```
Alternatively,
```ts
const trimmedMessage = `${message || ''}`.trim();
```

The function signature also indicates that `message` can't be `null` or `undefined`. Should it include those?

Reply:
``${message || ''}`.trim();` will swallow values like `0` or `false`. Even though the signature doesn't allow these values, we have some cases where the values from the view get passed straight into this method.